src/internal/io.js

```diff
@@ -241,3 +241,26 @@ export const asEffect = {
   getContext: createAsEffectType(GET_CONTEXT),
   setContext: createAsEffectType(SET_CONTEXT),
 }
+
+const nullOrWhat = (what, obj) => obj && (!what ? obj : obj[what])
+
+const _get = (what, ...args) => effect =>
+  args.reduce((maybe, type) => maybe || nullOrWhat(what, asEffect[type](effect)), null)
+
+export const inspection = {
```

Comment:
in general it's way better not to export objects, when they expose only static methods, as exporting objects in most cases prevents tree-shaking